common/scala/build.gradle

```diff
@@ -21,7 +21,7 @@ dependencies {
 
```

Comment:
Looks like spray-client is used by HttpClient.scala, but nothing uses the HttpClient code, so I deleted it.